project/plugins.sbt

```diff
@@ -23,6 +23,7 @@ libraryDependencies ++= Seq(
   "org.eclipse.jgit" % "org.eclipse.jgit" % "4.11.9.201909030838-r",
   "org.slf4j" % "slf4j-nop" % "1.7.36",
   "com.googlecode.java-diff-utils" % "diffutils" % "1.3.0",
+  //"io.github.java-diff-utils" % "java-diff-utils" % "4.11",
```

Comment:
remove?

Reply:
oops, I see the comment was for the new jar, and the question was, Is the other jar needed here for some reason?

Reply:
yes, it's used in build.sbt by the incredibly fragile `scala/runtime` patch for `test/instrumented` which should be renamed `test/demented`. I couldn't remove an import at some point in the past because of this mechanism. I'll try to understand what to do now.